packages/combo-box/test/keyboard.test.js

```diff
@@ -213,6 +213,43 @@ describe('keyboard', () => {
         expect(keydownSpy.called).to.be.false;
       });
 
+      it('should propagate keyboard enter event after entering an unknown option when custom values are allowed', () => {
+        comboBox.allowCustomValue = true;
+        setInputValue(comboBox, 'foobar');
+        enterKeyDown(input);
+
+        const keydownSpy = sinon.spy();
+        document.addEventListener('keydown', keydownSpy);
+        enterKeyDown(input);
```

Comment:
Could also use this instead of listening on the document. Not a blocker though.

```js
      const event = keyboardEventFor('keydown', 13, [], 'Enter');
      const keydownSpy = sinon.spy(event, 'stopPropagation');
```

Reply:
Thanks! I extracted that into a function and updated all tests to use it. Also added a missing test to the auto-open disabled suite.